client/src/shared/icons/GridIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import type { FC } from 'react';
+
+interface GridIconProps {
+  height?: number,
+  width?: number,
+  children?: string
+}
+const GridIcon: FC<GridIconProps> = ({ height, width, children }: GridIconProps) => {
```

Comment:
Unused parameters: height, width, children
Sinced you added them make sure to include it in the component